Studios sessions (#433)
seqeralabs/docs: platform_versioned_docs/version-24.2/data_studios/managing.mdx

     1. Customize the following fields:
        - **Container template**: Select a data studio template from the dropdown list.

>         - **Name**

Reviewer: ```suggestion
        - **Session name**
```

          If you select the **Prebuilt container image** template, you cannot select **Install Conda packages** as these options are mutually exclusive.
          :::

>         - **Name**

Reviewer: ```suggestion
        - **Session name**
```

When defining a new Studio session, you can configure the **Mounted data** by selecting the custom data-link created by the previous steps.

## Modify session lifespan for all Studios

For users with the **Admin** role, the lifespan for all newly launched studio sessions is configurable for a workspace.

1. Select **Settings**, and then for the Studios card select **Edit settings**.
1. Specify the session lifespan:
   - If you want to allow sessions to run indefinitely (the default), select **Always keep the session running**

>    - If you want to sessions to automatically shutdown after a specific lifetime, select **Automatically stop the session after a predefined lifespan**, and then input a **Lifespan** in hours between 1-120 hours.

Reviewer 2: ```suggestion
   - If you want to sessions to automatically shut down after a specific lifetime, select **Automatically stop the session after a predefined lifespan**, and then input a **Lifespan** in hours between 1-120 hours.
```
